webapp/src/main/webapp/news/components/NewsAppItem.vue

```diff
@@ -65,7 +65,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
             small-font-size
             popover />
         </div>
-        <div v-if="!draftNews" class="newsDate pe-4">
+        <div class="newsDate pe-4">
```

Comment:
You should test this modification for a draft article not already posted, normally these informations should not be displayed in that case. (otherwise to be verified with the PO)

Reply:
FYI in order to distinguish between a draft article not already posted and another draft of a posted article, you can base on news.activities which will be null in the first case and not null in the second case